tele_giphy/game/giphy.py

```diff
@@ -3,16 +3,30 @@
 giphy_url = 'http://api.giphy.com/v1/gifs/'
 key = 'dc6zaTOxFJmzC'
 
-def random(tag='american psycho', api_key=key):
-    # http://api.giphy.com/v1/gifs/random?api_key=dc6zaTOxFJmzC&tag=american+psycho
+def gif_random(tag='american psycho', api_key=key):
+    #Default: http://api.giphy.com/v1/gifs/random?api_key=dc6zaTOxFJmzC&tag=american+psycho
+    
+    # Accounting for cases where only one param is passed
+    if not tag:
+        tag = 'american psycho'
+    if not api_key:
+        api_key = key
+    
     endpoint = giphy_url + 'random'
     params = {'api_key':api_key,'tag':tag}
     data = get(endpoint, params)
     return data
 
-def translate(string='leeroy', api_key=key):
-    # http://api.giphy.com/v1/gifs/translate?s=superman&api_key=dc6zaTOxFJmzC
+def gif_translate(string='leeroy', api_key=key):
+    #Default: http://api.giphy.com/v1/gifs/translate?s=leeroy&api_key=dc6zaTOxFJmzC
+    
+    # Accounting for cases where only one param is passed 
+    if not string:
+        string = 'leeroy'
+    if not api_key:
+        api_key = key
+    
```

Comment:
The default values are already set, there's no reason for these if-statements. You can send in just the key or just the string, the other one is still set.